### PR TITLE
added file picker (using file_picker_cross and flutter_platform_widgets)

### PR DESCRIPTION
**Pull request by CoolONEOfficial**

### CODE REVIEW

File: lib/widgets/picker_fields/card_settings_file_picker.dart [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+                contentWidget: Text(
+                  formattedValue,
+                  style: widget?.style ?? Theme.of(context).textTheme.subtitle1,
+                  textAlign: widget?.contentAlign ??
+                      CardSettings.of(context).contentAlign,
+                ),
+                style: CSWidgetStyle(icon: widget?.icon),
+              ),
+            ),
+    );
```
Comment (codegrue) [inline lib/widgets/picker_fields/card_settings_file_picker.dart:169]: Your code still refers to DatePicker in some of the names, please fix.

Comment (codegrue) [reply lib/widgets/picker_fields/card_settings_file_picker.dart:169]: Does this work with web?

**Comment by codegrue** [comment]
Nice work thank you!

File: lib/widgets/picker_fields/card_settings_file_picker.dart (hunk 1 above)
Comment (CoolONEOfficial) [reply lib/widgets/picker_fields/card_settings_file_picker.dart:169]: > Does this work with web?

Yep, check file_picker_cross docs.

Comment (codegrue) [reply lib/widgets/picker_fields/card_settings_file_picker.dart:169]: FYI I renamed the methods

**Pull request merged**